### ISSUE TITLE
Thematic Structuring of Collections and Display of Vector Tile URLs in BBOX

### ISSUE THREAD

**Issue by FabianRechsteiner** [bbox-services/bbox#81]
**Description:**
Currently, all tables from the PostGIS database are displayed in Collections. I would like to discuss the possibility of thematically structuring these collections. For example:

- We have various tables in the `amtliche_vermessung` schema that should be listed under a theme called "Amtliche Vermessung" in BBOX collections.

- The table `solarkataster`, on the other hand, should be found under the theme "Energie".

My goal is to use BBOX to provide an API through which all geographic data of the city of Winterthur can be accessed. The idea is to have a catalog that displays different themes and datasets, with descriptions, information, and metadata linked to these themes and datasets.

Currently, the collections only display the items. It would also be beneficial to display the URL for Vector Tiles if available. Additionally, some collections might be combined into a single tileset, and it would be useful to show these as well.

I’m interested in how other users handle the organization of large datasets and if there are suggestions for effectively structuring and presenting a large number of data sources. Are there best practices or features that could help achieve this?

**Comment by pka** [comment]
Maybe [STAC](https://stacspec.org/en/tutorials/intro-to-stac/) Catalogs could be a way for grouping topics? (see also #61, @kdesjard ?)

**Comment by kdesjard** [comment]
Indeed, I have a fork that I've been meaning to get packaged for upstream.

You can see here how we are using it:

https://radiantearth.github.io/stac-browser/#/external/www.eodms-sgdot.nrcan-rncan.gc.ca/ogc/catalog.json

**Comment by pka** [comment]
> Indeed, I have a fork that I've been meaning to get packaged for upstream.
> 
> You can see here how we are using it:
> 
> https://radiantearth.github.io/stac-browser/#/external/www.eodms-sgdot.nrcan-rncan.gc.ca/ogc/catalog.json

We're very interested in your Stac implementation! I found two branches:
* https://github.com/kdesjard/bbox/tree/stac_compliance
* https://github.com/kdesjard/bbox/tree/stac_searching

Are you fine with merging them into upstream bbox? Are there dependencies between them?

**Comment by kdesjard** [comment]
Sorry, missed your previous message.  I need to work on some small fixes for link hierarchy and then I'll have to rebase and create PR's.  I'll try to get this done within the year.
